test/WebSites/RazorPagesWebSite/Pages/Redirects/RedirectToSibling.cshtml

```diff
@@ -8,4 +8,6 @@
     public IActionResult OnGetRedirectToSubDir() => RedirectToPage("SubDir/SubDirPage");
 
     public IActionResult OnGetRedirectToParent() => RedirectToPage("../Conventions/AuthFolder/Index");
+
+    public IActionResult OnGetRedirectToDotSlash() => RedirectToPage("./SubDir/SubDirPage");
```

Comment:
Double check my thinking that `/Pages/Redirects/SubDir/SubDirPage` is the correct outcome of this (as tested above). This seems correct to me but some instinct keeps wanting the correct outcome to be `/Pages/Redirects/RedirectToSibling/SubDir/SubDirPage`.